File: 백준/Bronze/17363. 우유가 넘어지면？/우유가 넘어지면？.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : 'run/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

const [n, m] = input[0].split(' ').map(Number);
const box = input.slice(1).map((row) => row.split(''));

const charMap = {
  '^': '<',
  '>': '^',
  v: '>',
  '<': 'v',
  '-': '|',
  '|': '-',
  '/': '\\',
  '\\': '/',
};

const rotate = [];
for (let i = m - 1; i >= 0; i--) {
  const row = [];
  for (let j = 0; j < n; j++) {
    const c = box[j][i];
    row.push(charMap[c] || c);
  }
  rotate.push(row);
}

for (const row of rotate) {
  console.log(row.join(''));
}

File: 백준/Bronze/17363. 우유가 넘어지면？/우유가 넘어지면？.py
d = {'.': 46, 'O': 79, '-': 124, '|': 45, '/': 92, '\\': 47, '^': 60, '<': 118, 'v': 62, '>': 94}

n, m = map(int, input().split())
box = [input() for _ in range(n)]
rotate = []

for i in range(m - 1, -1, -1):
     row = ['%c' % d[box[j][i]] for j in range(n)]
     rotate.append(row)

for row in rotate:
     print(''.join(row))